project_euler/problem_102/sol102.py

```diff
@@ -0,0 +1,94 @@
+"""Project Euler Problem 102: https://projecteuler.net/problem=80
+
+"It is well known that if the square root of a natural number is not an
+integer, then it is irrational. The decimal expansion of such square
+roots is infinite without any repeating pattern at all.
+
+"The square root of two is 1.41421356237309504880..., and the digital
+sum of the first one hundred decimal digits is 475.
+
+"For the first one hundred natural numbers, find the total of the
+digital sums of the first one hundred decimal digits for all the
+irrational square roots."
+
+To solve this problem, create vectors from (0,0) to each of the points
+of the triangle. Then, add the angles between all the vectors. If the
+triangle includes the origin, the angles will add to 2*pi
+radians. Otherwise, they will add to less.
+
+References:
+ - https://stackoverflow.com/questions/2049582/
+
+"""
+import math
+import os
+
+
+class vector:
+    """
+    A simple vector class
+    """
+
+    def __init__(self, x: float, y: float) -> None:
+        """
+        Initialize a vector with x and y coordinates.
+
+        >>> isinstance(vector(1,2), vector)
+        True
+        """
+
+        self.x = x
+        self.y = y
+
+    def len(self) -> float:
```

Comment:
```suggestion
    def length(self) -> float:
```
`len` is a built-in function

Reply:
Done.